code/update-rnet.R

```diff
@@ -0,0 +1,81 @@
+library(pct)
+library(stplanr)
+library(tidyverse)
+library(sf)
+
+# readRDS(url("https://github.com/npct/pct-outputs-national/blob/master/commute/lsoa/rnet_all.Rds"))
+rnet = pct::get_pct_rnet(region = "avon") # LSOA route network
+# mapview(rnet["govtarget_slc"])
+
+r_cyipt = readRDS("rsf.Rds")
+r_cyipt = r_cyipt[r_cyipt$region == "Bristol",]
+
+# st_crs(rnet)
+# st_crs(r_cyipt)
+
+dim(rnet) # 13829 this only has streets which are part of the LSOA route network
+dim(r_cyipt) # 104805 this has every street
+
+rnet_buff = geo_buffer(shp = rnet, dist = 10)
+# mapview(rnet_buff["govtarget_slc"])
+
+r_cyipt_joined = st_join(r_cyipt, rnet_buff, join = st_within)
```

Comment:
One question: did you try this with the `largest` argument set to true, i.e.: 

```r
 r_cyipt_joined = st_join(r_cyipt, rnet_buff, join = st_within, largest = TRUE)
```

?